### PR TITLE
Add 1.18.2 & Java 17

### PR DESCRIPTION
**Pull request by Blocky38**
Script is tested on Windows. Linux should work to only thing could be the Java17-URL.

### CODE REVIEW

File: setup.sh [hunk 1]
```diff
@@ -114,9 +114,44 @@ mkdir 1.16.3
...
+cd java17
+echo Downloading Java-17 ...
+sleep 3
+
+if [[ "$SYSTEM" == *"MINGW64_NT"* ]];
+then
+PATH_JAVA_17="jdk-17_windows-x64_bin"
+curl -O https://download.oracle.com/java/17/latest/$PATH_JAVA_17.zip
+else
+PATH_JAVA_17="openjdk-16.0.2_linux-x64_bin"
```
Comment (TheBlackEntity) [inline setup.sh:130]: Should be PATH_JAVA_17="jdk-17_linux-x64"

**Comment by TheBlackEntity** [comment]
I'll test the file tomorrow.
Thanks for the pull request :)

**Comment by Blocky38** [comment]
> I'll test the file tomorrow. Thanks for the pull request :)

I will push a correction in some minutes. At the moment I am testing it out on my ubuntu vm.

**Comment by Blocky38** [comment]
Is working on Ubuntu/Kali Linux

**Pull request merged**